app/views/kudos/index.html.erb

```diff
@@ -1,4 +1,10 @@
 <h1>Kudos listing page</h1>
+<p> <%if current_employee.number_of_available_kudos > 0%>
+      <%=button_to "Create New Kudo", new_kudo_path , method: :get%>
+    <% else %>
+      <p><h6> You have used all your kudos</h6></p>
+    <%end%>
```

Comment:
to zbyt małe zabezpieczenie, co jeżeli user wejdzie w stronę do tworzenia nowych kudosów nie mając available_kudosów, wpisując adres strony z palca w przeglądarkę